Release/ConEmu/ConEmu.l10n

```diff
@@ -233,13 +233,13 @@
       "id": 2278 }
     ,
     "cbClinkWebPage": {
-      "en": "Open clink's web page https://mridgers.github.io/clink/ in your browser",
```

Comment:
These should be changed in https://www.transifex.com/conemu/conemu-sources/dashboard/
Strings are downloaded automatically during build.

Reply:
Oh, transifex has no two-way sync? OK, I will apply for all languages to propse the new URL.

Reply:
OK, I don't understand the system.

https://github.com/Maximus5/ConEmu/blob/20a1dacd2459b7c4c0fad224bc0488ae89cc82f5/Release/ConEmu/ConEmu.l10n#L265

shows 

    "de": "Öffnet clinks Website https://mridgers.github.io/clink/ im Browser",

But transifex shows

![grafik](https://user-images.githubusercontent.com/1366654/105774962-47bfff00-5f66-11eb-8626-f0586768fd1c.png)

Think, there is just a sync missing.

Reply:
I've already updated transifex manually.

There is a script to download translations from transifex.
The two-way sync would be rather complicated, even if possible, isn't it?